packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -3146,10 +3146,10 @@ export default abstract class SqlIntegration
 
     const sqlQuery = `
       SELECT
-        ${userIdColumn} as user_id,
+        ${hasUserId ? `${userIdColumn}, ` : ""}
```

Comment:
Do we want to keep the alis (`as user_id`) here?

Reply:
ah - yes, good catch. Updated for metrics and fact tables.